src/extender.cpp

```diff
@@ -420,7 +420,7 @@ namespace Sass {
 
     // for (const Extension extension : oldExtensions) {
     for (size_t i = 0; i < oldExtensions.size(); i += 1) {
```

Comment:
Perhaps this should just be:

```c++
for (const Extension& extension : oldExtension) {  
```

or `Extension extension : ...` if it needs to be copied (doesn't look like it needs to though)

Reply:
That's exactly what I tried to achieve two lines below ...